host/hackrf-tools/src/hackrf_info.c

```diff
@@ -26,126 +26,144 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-int main(void)
-{
+int main(void) {
 	int result = HACKRF_SUCCESS;
 	uint8_t board_id = BOARD_ID_INVALID;
 	char version[255 + 1];
 	uint16_t usb_version;
 	read_partid_serialno_t read_partid_serialno;
 	uint8_t operacakes[8];
-	hackrf_device_list_t *list;
+	hackrf_device_list_t* list;
 	hackrf_device* device;
 	int i, j;
 
 	result = hackrf_init();
 	if (result != HACKRF_SUCCESS) {
-		fprintf(stderr, "hackrf_init() failed: %s (%d)\n",
-				hackrf_error_name(result), result);
+		fprintf(stderr, "hackrf_init() failed: %s (%d)\n", hackrf_error_name(result), result
+		);
```

Comment:
The automatic reformatting causes plenty of these hanging parens.

Reply:
Fixed by no longer using the `BlockIndent` setting.